### DISCUSSION TITLE
How to use vector graphics

### DISCUSSION THREAD

**Discussion by goffchris** [backface/turtlestitch#137] [Q&A]
I have seen the examples of vector graphics being included in designs, but can't find any tutorials or documentation on how to use them. Please point me in the right direction.

**Comment by chmod4711** [comment]
Hi Chris! There is no public documentation available at that time unfortunately. We're in contact with a user who recently posted a project. I'm going to reach out to him to let us know about his process. 
We're currently working on the relaunch of the TurtleStitch cloud. Documentation in terms of coding refernce and also making things (physical) is part of it.
